pkg/cloud/tags.go

```diff
@@ -34,71 +39,62 @@ type TagIface interface {
 type ResourceType string
 
 const (
-	clusterTagNamePrefix               = "CAPC_cluster_"
-	createdByCAPCTagName               = "created_by_CAPC"
+	ClusterTagNamePrefix               = "CAPC_cluster_"
+	CreatedByCAPCTagName               = "created_by_CAPC"
```

Comment:
Should these start with upper case letters?  Are you exporting them?

Reply:
I use them in tests, but I could also see us using them in the controllers at some point.

Reply:
Good reason.  👍